README.md

```diff
@@ -1,2 +1,14 @@
 # VideoReviewConsoleApp
 Create Video Reviews in Content Moderator
+## Getting Started
+### For a Single Video File
+Build and run the code, a console should be present with the following messege:
+#### Enter the fully qualified local path for Uploading the video :
+Format of the local path can be in quotes i.e. "C:\{path}" or without quotes i.e. C:\{path}
+### Transcription Option
+Choose y for transcription n for no transcription
+#### Transcription Screentext Tagging
+In order for transcription screentext result to work, a user must add Adult Text(at), Racy Text(rt), and Offensive Text(ot) tags to the team via Setting->Tags page.
```

Comment:
The transcription is screened for Profanity using Content Moderator Text API. This API screens the text for known profanity terms in 100+ languages. For english, this API also returns classification scores for Adult, Racy and Offensive categories.

Video frames whose transcript are flagged by the Text Screen API can be automatically flagged in the video review.

To enable this, add the following Tags to your Content Moderator Team:
Name: Adult Text
Short Code: at

Name: Racy Text
Short Code: rt

Name: Offensive Text
Short Code: ot

--- Or better you can make this a table